charts/airbyte/templates/webapp/service.yaml

```
@@ -3,6 +3,10 @@ apiVersion: v1
 kind: Service
 metadata:
   name: {{ include "common.names.fullname" . }}-webapp
+  {{- if .Values.webapp.service.annotations }}
+  annotations:
+    {{- include "common.tplvalues.render" (dict "value" .Values.webapp.service.annotations "context" $) | nindent 4 }}
+  {{- end }}
```

```suggestion
  {{- with .Values.webapp.service.annotations }}
  annotations:
    {{- toYaml . | nindent 4 }}
  {{- end }}
```

Does something prevents you from following the same syntax as used in `ingress.yaml` to create indentations? I'd prefer both file to use the same syntax for consistency.